collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -192,7 +182,7 @@ public void onClick(View v) {
 
             // indicating whether or not select-all is on or off.
```

Comment:
I believe this made sense when "toggle" meant a true alternation but isn't relevant anymore. The whole conditional and comment should be removed.